cscs-checks/apps/amber/amber_check.py

```diff
@@ -131,29 +98,41 @@ def set_perf_patterns(self):
                                              float, item=1)
         }
 
+    @run_after('init')
```

Comment:
Hooks must show up in chronological order. post init hooks run before the setup stage.